magefile.go

```diff
@@ -391,20 +420,28 @@ type Build mg.Namespace
 
 // Build all PKO binaries for the architecture of this machine.
 func (Build) Binaries() {
-	mg.Deps(
-		mg.F(Builder.Cmd, "package-operator-manager", runtime.GOOS, runtime.GOARCH),
-		mg.F(Builder.Cmd, "remote-phase-manager", runtime.GOOS, runtime.GOARCH),
-		mg.F(Builder.Cmd, "kubectl-package", runtime.GOOS, runtime.GOARCH),
-		mg.F(Builder.Cmd, "mage", "", ""),
-	)
+	targets := []interface{}{mg.F(Builder.Cmd, "mage", "", "")}
+	for name := range commands {
+		targets = append(targets, mg.F(Builder.Cmd, name, nativeArch.OS, nativeArch.Arch))
+	}
+
+	mg.Deps(targets...)
 }
 
-func (Build) MultiArchBinaries() {
-	targets := []mg.Fn{mg.F(Build.Binaries)}
-	for _, tuple := range cliExtraArch {
-		targets = append(targets, mg.F(Builder.Cmd, "kubectl-package", tuple[0], tuple[1]))
+func (Build) ReleaseBinaries() {
+	targets := []interface{}{}
+	for name, cmd := range commands {
+		for _, arch := range cmd.ReleaseArchitectures {
+			targets = append(targets, mg.F(Builder.Cmd, name, arch.OS, arch.Arch))
+		}
+	}
+	mg.Deps(targets...)
+
+	for name, cmd := range commands {
+		for _, arch := range cmd.ReleaseArchitectures {
+			must(sh.Copy(fmt.Sprintf("%s.%s.%s", name, arch.OS, arch.Arch), binaryPath(name, arch)))
```

Comment:
```suggestion
			must(sh.Copy(fmt.Sprintf("%s_%s_%s", name, arch.OS, arch.Arch), binaryPath(name, arch)))
```

Isn't underscore more common than separating by dots?
e.g. binaries here: https://github.com/kcp-dev/kcp/releases/tag/v0.9.1

Reply:
since we already use "-" within the command name it is not a good one. "kubectl-package-linux-amd64".

But if you want that I can implement it